src/actions/folderActions.js

```diff
@@ -98,7 +98,7 @@ export const createFolder = data => {
       })
   }
 }
-export const editFolder = (id, data) => {
+export const editFolder = (id, data, callback) => {
```

Comment:
always add default value to callback as it is not necessary to call it.
if it is not given in the parameters then this will give error

Reply:
I have changed it ... but it was not giving error without default mentioned as well.